sources/3.0.0/sections/04-data_content_struct.adoc

```diff
@@ -157,12 +157,12 @@ The class *CV_GridEnvelope* provides the grid coordinate values for the diametri
 [level=7]
 ===== low
 
-The attribute _low_ shall be the minimal coordinate values for all grid points within the envelope. For this specification this represents the Southwestern coordinate.
+The attribute _low_ shall be the minimal coordinate values for all grid points within the envelope. For this specification this represents the Southwestern corner coordinate of the southwesternmost grid cell.
 
 [level=7]
 ===== high
 
-The attribute _high_ shall be the maximal coordinate values for all grid points within the envelope. For this specification this represents the Northeastern coordinate.
+The attribute _high_ shall be the maximal coordinate values for all grid points within the envelope. For this specification this represents the Northeastern corner coordinate of the northeasternmost grid cell.
```

Comment:
A clarification of some sort might be useful. As grid points are in the center of the cell this would be grid point coordinate plus half grid spacing in both X- and Y-directions.

Reply:
Marking this as resolved since this is discussed above.